### PR TITLE
Embed afforms & search displays on contact summary screen as blocks and tabs

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
This enables the creation of blocks and tabs on the contact summary screen, using Afform & SearchKit.

Forms containing a search display will automatically receive the id of the contact being viewed as a filter. Tabs will automatically show a counter in the tab of the number of rows (like core tabs do).

The blocks/tabs will be placed in default positions, but can be rearranged with the Contact Layout Editor extension.

**Embed Afform Like This:**
image

**This is the Contribution tab from core:**
image

**And here is the one I made with SearchKit:**
image

Technical Details
----------------------------------------
One of the technical challenges was the ability to boostrap Angular more than once per page. This has been solved by #19922.

### CODE REVIEW

**Comment by mattwire** [comment]
@colemanw This sounds great! Just a note that we need to make sure that `selectedChild=` for selecting tabs via URL still works after this change.

**Comment by colemanw** [comment]
> @colemanw This sounds great! Just a note that we need to make sure that `selectedChild=` for selecting tabs via URL still works after this change.

Yep, just tried it and it works. Just use the name of the afform as the value of selectedChild.

**Comment by eileenmcnaughton** [comment]
I gave this a quick spin and it's VERY COOL

I'm trying to think of a great use case to demo with it

**Comment by eileenmcnaughton** [comment]
@seamuslee001 I'd be happy to merge this based on my testing - but I think you want to do some too?

**Comment by colemanw** [comment]
Ok. The other PR has been merged & I've rebased this one.

File: ext/afform/core/afform.php [hunk 1]
```diff
@@ -174,16 +174,92 @@ function afform_civicrm_managed(&$entities) {
...
  *
- * Generate a list of case-types.
+ * Adds afforms as contact summary tabs.
+ */
+function afform_civicrm_tabset($tabsetName, &$tabs, $context) {
+  if ($tabsetName !== 'civicrm/contact/view') {
+    return;
+  }
+  $scanner = \Civi::service('afform_scanner');
+  $weight = 111;
```
Comment (seamuslee001) [inline ext/afform/core/afform.php:186]: @colemanw is there any reasoning behind this specific weight? are we not able to just get the max weight of the current tabs?

Comment (colemanw) [reply ext/afform/core/afform.php:186]: Well, we generally want the "ChangeLog" tab to be last and that has a weight of 120, so this puts the afform tabs after core tabs but before changelog.

The default positions for tabs and blocks can be configured with the Contact Layout Editor, which I think is the right answer for anyone wanting to change it.

Comment (eileenmcnaughton) [reply ext/afform/core/afform.php:186]: Also 111 is the emergency number in NZ

**Comment by seamuslee001** [comment]
I tested this and worked well merging

**Comment by eileenmcnaughton** [comment]
woohoo

**Pull request merged**